aws/table_aws_vpc_security_group_rule.go

```diff
@@ -106,18 +106,18 @@ func tableAwsVpcSecurityGroupRule(_ context.Context) *plugin.Table {
 			},
 			{
 				Name:        "cidr_ip",
-				Description: "[DEPRECATED] This column has been deprecated and will be removed in a future release, use cidr_ipv4 instead. The IPv4 CIDR range. It can be either a CIDR range or a source security group, not both. A single IPv4 address is denoted by /32 prefix length.",
```

Comment:
This is an older column, based on the way we populate it, could it ever be a source security group, or was the description wrong back then too?